cmd/service/utils.go

```diff
@@ -31,8 +35,32 @@ func handleError(err error) {
 	}
 }
 
+func gitClone(url string) (path string, err error) {
+	tmpFile := "/tmp/mesg-templates"
```

Comment:
This is very ugly to put the path in a root folder like this.
It's not even deleted at the end.